tinydrive/tinydrive-api/standalone.md

```diff
@@ -0,0 +1,282 @@
+
+## tinydrive.start
+
+Start is similar to pick and browse, the main difference is that method doesn't have a way of closing the dialog. This can be useful when you want to lauch Tiny Drive from an url where there is no application to insert the files into. This might be useful when you want to launch Tiny Drive from an url but not part of a bigger application. For a complete list of available settings to pass in to this method check the [Picker settings](#pickersettings) section in this page.
+
+### Example
+
+```js
+tinydrive.start({
+  token_provider: '/your-local/jwt-provider'
+});
+```
+
+## Generic settings
+
+These settings can be passed to any of the the standalone api methods.
+
+### `token_provider`
+
+This setting could take one of the following two forms:
+
+* A URL to a page that takes an HTTP JSON POST request and produces a JSON structure with a valid JWT. It uses a POST request to avoid caching by browsers and proxies.
+* A function that provides the same token through a callback. This allows making an HTTP request in any desired format. The provider function is a function that has a success and failure callback where the success takes an object with a token property containing the JWT, and the failure callback takes a string to present as an error message if the token could not be produced.
+
+For more information on how to create these tokens, refer to the [JWT authentication guide]({{site.baseurl}}/tinydrive/jwt-authentication/) or try one of the [starter projects]({{site.baseurl}}/tinydrive/getting-started/#starterprojects).
+
+**Type:** `String` or `Function`
+
+**Required:** yes
+
+#### Example Using a JWT Provider URL
+
+```js
+tinydrive.pick({
+  token_provider: '/jwt' // this can be a page or endpoint like this
+});
+```
+
+#### Example Using a JWT Provider Callback
+
+```js
+tinydrive.pick({
+  token_provider: function (success, failure) {
+     success({ token: 'jwt-token' });
+     // failure('Could not create a jwt token')
+  }
+}).then((result) => {
+  console.log(result.files);
+});
+```
+
+## Picker settings
+
+These settings are available for the [tinydrive.pick](#tinydrivepick) / [tinydrive.browse](#tinydrivebrowse) / [tinydrive.start](#tinydrivestart) api methods.
+
+### `dropbox_app_key`
+
+This setting enables specifying the Dropbox API key for integrating dropbox into Tiny Drive. For more information on how you obtain this key, refer to the [Dropbox integration guide]({{site.baseurl}}/tinydrive/integrations/dropbox-integration/).
+
+**Type:** `string`
+
+#### Example
+
+```js
+tinydrive.pick({
+  dropbox_app_key: '<your dropbox app key>'
+}).then((result) => {
+  console.log(result.files);
+});
+```
+
+### `filetypes`
+
+This setting enables to restrict what types of files you want do display based on [file type]({{site.baseurl}}/tinydrive/introduction/#filetypes) categories. For example if your app needs to insert images only then you can specify `['image']` in the file types array.
```

Comment:
Maybe instead of `enables to restriction what types`, use `enables restricting what types`? and similarly the same in plugin.md.